src/student/components/sessionDashboardView/SessionDashboardView.tsx

```diff
@@ -25,7 +25,7 @@ export function SessionDashboardView() {
             overflow: "hidden",
         },
         backdrop: {
-            zIndex: theme.zIndex.drawer + 1,
+            zIndex: theme.zIndex.drawer + 100,
```

Comment:
why 100 instead of 1?

Reply:
Tried to fix the scroll visible from feedback view